### DISCUSSION TITLE
Welcome to manylinux Discussions!

### DISCUSSION THREAD

**Discussion by mayeut** [pypa/manylinux#1401] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Trizkitz** [comment]
Hi 👋 everyone! I'm still kind of new to git  but have been using it for an extra log-in security level. also I have been looking around from time to time checking out the github app together comfortable with it because coding and algorithms are complicated and watching YouTube videos on github were a bit confusing at times..I'm Canadian 🇨🇦 single guy living with 2 cats in my mid 30's looking to make some cash money in the comfort of my own residence and with an intriguing community on the same topic able to help each other solve complex problems acquiring a digital platform of some sort for the world to utilize. feel free anyone to hit it in the comments or if there is a way to Direct message then feel free! I haven't got much done with pulls or anything but I do get offers....
